### PR TITLE
Add sort feature for list and Help table

### LINKED ISSUE #35: Sort by price
As a pharmacist, I want to be able to sort medication by price so that if a customer asks for the cheapest one I can recommend him

### PR DESCRIPTION
**Pull request by alvintan01**
Fix #35 #36

### CODE REVIEW

File: src/main/java/parser/StockValidator.java [hunk 1]
```diff
@@ -163,9 +163,11 @@ public static boolean isValidMaxQuantity(Ui ui, String maxQuantityString) {
      */
     public static boolean isValidColumn(Ui ui, String columnName) {
         String[] columnAlias = new String[]{CommandParameters.STOCK_ID, CommandParameters.NAME, CommandParameters.PRICE,
-            CommandParameters.QUANTITY, CommandParameters.EXPIRY_DATE, CommandParameters.DESCRIPTION,
-            CommandParameters.MAX_QUANTITY};
-        if (Arrays.asList(Stock.COLUMNS).contains(columnName) || Arrays.asList(columnAlias).contains(columnName)) {
+                                            CommandParameters.QUANTITY, CommandParameters.EXPIRY_DATE,
```
Comment (RemusTeo) [inline src/main/java/parser/StockValidator.java:166]: Is this indentation correct? I think indentation for wrapped lines should only be 8 spaces more than parent line.
https://se-education.org/guides/conventions/java/basic.html

File: src/main/java/ui/Ui.java [hunk 2]
```diff
@@ -121,7 +122,7 @@ public void printMedicines(ArrayList<Medicine> medicines) {
         String maxQuantityFormat = "%1$-" + maxQuantityWidth + "s | ";
 
         String[] formats = {idFormat, nameFormat, priceFormat, quantityFormat, expiryFormat, descriptionFormat,
-            maxQuantityFormat};
+                            maxQuantityFormat};
```
Comment (RemusTeo) [inline src/main/java/ui/Ui.java:125]: Is this indentation correct? I think indentation for wrapped lines should only be 8 spaces more than parent line.
https://se-education.org/guides/conventions/java/basic.html

File: src/main/java/ui/Ui.java [hunk 3]
```diff
@@ -109,7 +110,7 @@ public void printMedicines(ArrayList<Medicine> medicines) {
         }
 
         int[] columnWidths = {idWidth, nameWidth, priceWidth, quantityWidth, expiryWidth, descriptionWidth,
-            maxQuantityWidth};
+                              maxQuantityWidth};
```
[the remark this answers is not in the document]
Comment (alvintan01) [reply src/main/java/ui/Ui.java]: For now, we will use 12 spaces although the original indentation of 8 is correct by coding standards.

File: src/main/java/ui/Ui.java (hunk 2 above)
Comment (alvintan01) [reply src/main/java/ui/Ui.java]: For now, we will use 12 spaces although the original indentation of 8 is correct by coding standards.

**Pull request merged**